.changeset/mighty-geese-cheer.md

```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/icon": patch
```

Comment:
Please change `patch` to `major`.

Reply:
I changed it. [cb8ef85](https://github.com/yamada-ui/yamada-ui/pull/2537/commits/cb8ef85aa5fda25cf1edebab9ce196ecb6b8216d)